package.json

```diff
@@ -29,7 +29,7 @@
   },
   "homepage": "https://github.com/Eppo-exp/node-server-sdk#readme",
   "dependencies": {
-    "@eppo/js-client-sdk-common": "4.8.0-alpha.1"
+    "@eppo/js-client-sdk-common": "4.8.1"
```

Comment:
Are you prepping for release in this PR as well or with a follow up? this package's version will need to be updated.

Reply:
ah good call, totally forgot that part. thanks!